### PR TITLE
ROC-4699 Claim check-and-send route tests

### PR DESCRIPTION
**Pull request by shrivsksavit**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-4699

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/test/features/claim/routes/check-and-send.ts [hunk 1]
```diff
@@ -89,6 +89,8 @@ describe('Claim issue: check and send page', () => {
           .expect(res => expect(res).to.be.successful.withText('<a href="/claim/evidence" class="bold">Change <span class="visuallyhidden">your evidence (optional)</span></a>'))
           .expect(res => expect(res).to.be.successful.withText('input id="signedtrue" type="checkbox" name="signed" value="true"'))
           .expect(res => expect(res).to.be.successful.withText('input type="submit" class="button"'))
+          .expect(res => expect(res).to.be.successful.withText('Statement of truth'))
```
[the remark this answers is not in the document]
Comment (shrivsksavit) [reply src/test/features/claim/routes/check-and-send.ts]: updated please have a look

File: src/test/features/claim/routes/check-and-send.ts [hunk 2]
```diff
@@ -87,270 +87,97 @@ describe('Claim issue: check and send page', () => {
...
+              })
+            feesServiceMock.resolveCalculateIssueFee()
+
+            await request(app)
+              .get(ClaimPaths.checkAndSendPage.uri)
+              .set('Cookie', `${cookieName}=ABC`)
+              .expect(res => expect(res).to.be.successful.withText('<a href="/claim/claimant-sole-trader-details" class="bold">Change <span class="visuallyhidden">full name</span></a>'))
+              .expect(res => expect(res).to.be.successful.withText('Business name'))
+              .expect(res => expect(res).to.be.successful.withText('Trading as SoleTrader Ltd.'))
+              .expect(res => expect(res).to.be.successful.withText('Statement of truth'))
```
Comment (KiranVarma02) [inline src/test/features/claim/routes/check-and-send.ts]: I am ok with all mostly based on what we have done in other code but I am not sure how useful it is to test for example same 'Statement of truth' text and same 'I believe that the facts stated in this claim are true' text for each party (4 times). It's good coverage i guess but there will be lot to maintain later. @shrivsksavit I understand you have done based on existing examples but let's get other opinions and see what they think. @robertparkinson @simad

Comment (robertparkinson) [reply src/test/features/claim/routes/check-and-send.ts]: I agree with Kiran, checking for the same content 4 times doesn't seem appropriate. The code that displays the claimant and the defendant are not linked. A test that checks that the correct content shows up when each party type is either a claimant or defendant would be sufficient coverage.
The tests you previously were doing that. they were checking that both combinations of content were appearing which seems a more valid test in my mind.

Comment (shrivsksavit) [reply src/test/features/claim/routes/check-and-send.ts]: accepted and updated please have a look

File: src/test/features/claim/routes/check-and-send.ts [hunk 3]
```diff
@@ -87,13 +88,15 @@ describe('Claim issue: check and send page', () => {
           .expect(res => expect(res).to.be.successful.withText('<a href="/claim/reason" class="bold">Change <span class="visuallyhidden">why you believe you’re owed the money:</span></a>'))
```
Comment (andrewwa-kainos) [inline src/test/features/claim/routes/check-and-send.ts]: `.withText(...)` accepts a vararg of strings, so you can just comma-separate them

Comment (shrivsksavit) [reply src/test/features/claim/routes/check-and-send.ts]: updated please have a look

Comment (andrewwa-kainos) [reply src/test/features/claim/routes/check-and-send.ts]: LGTM

**Pull request merged**